setuptools/tests/test_egg_info.py

```diff
@@ -567,6 +567,204 @@ def test_setup_cfg_license_file(
             assert 'LICENSE' not in sources_text
             assert 'INVALID_LICENSE' not in sources_text # for invalid license test
 
+    @pytest.mark.parametrize("files, incl_licenses, excl_licenses", [
+        ({
+            'setup.cfg': DALS("""
+                              [metadata]
+                              license_files =
+                                  LICENSE-ABC
+                                  LICENSE-XYZ
+                              """),
+            'LICENSE-ABC': DALS("ABC license"),
```

Comment:
`DALS()` is only needed for multi-line strings.

Reply:
Fixed